server/errorHandler.test.ts

```diff
@@ -52,4 +52,12 @@ describe('GET 500', () => {
         expect(res.text).toContain('Please try again later.')
       })
   })
+
+  it('should return 500 status', () => {
+    return request(appWithAllRoutes({ production: true }))
+      .get('/')
+      .expect(res => {
+        expect(res.statusCode).toEqual(500)
+      })
+  })
```

Comment:
I appreciate this test works, and that 500 errors are hard to simulate, but it is a bit confusing that a request for what looks like the homepage returns a 500 error